### ISSUE TITLE
TODO: add config option to capture mouse events

### ISSUE THREAD

**Issue by ihadeed** [zyra/ionic2-super-tabs#21]
At the moment, only touch events are captured, which is ideal. Some developers might want the slides to listen to mouse events as well (treat them as touch events). The `config` input should accept an option to enable this.

**Comment by benharpers** [comment]
Yes, I was just running into this issue trying to debug stuff and being stuck only using a real device or a simulator, would be a great addition.
